### PR TITLE
chore: rename sieve security to sieve filter

### PR DESCRIPTION
**Pull request by kesselb**
- Rename "Enable sieve security" to "Enable sieve filter"
- Use the translation function
- Add a small introduction for the feature

image

### CODE REVIEW

File: src/components/SieveAccountForm.vue [hunk 1]
```diff
@@ -4,9 +4,12 @@
 -->
 <template>
 	<form id="sieve-form">
+		<p>
+			{{ t('mail', 'Sieve is a powerful language for writing filters for your mailbox. You can control the sieve scripts from Nextcloud Mail if your email service supports it.')}}
```
Comment (ChristophWurst) [inline src/components/SieveAccountForm.vue:8]:
```suggestion
			{{ t('mail', 'Sieve is a powerful language for writing filters for your mailbox. You can control the sieve scripts from {product} Mail if your email service supports it.', { product: OCA.theme.????? })}}
```

we use it in other places too. forgot the exact variable ;-)

Comment (ChristophWurst) [reply src/components/SieveAccountForm.vue:8]: https://github.com/nextcloud/mail/blob/629a9fa693648d9414ecc8cb0fc3f315305dbba8/src/components/NoMessageSelected.vue#L33

Comment (kesselb) [reply src/components/SieveAccountForm.vue]: I like the idea :+1: 

https://github.com/nextcloud/mail/pull/10100/commits/f17aba480a09e31d8b58f06a09861b4a00618842 did not work, even if `window?.OC?.theme?.name` evaluates to "Testcloud". I assume that OC.theme is not populated when the component is rendered, but didn't look at it properly. 

Pushed another fixup to use a wording without Nextcloud. Let me know if you want me to take a look later to include the instance name or product name, I will create an issue then.

Comment (ChristophWurst) [reply src/components/SieveAccountForm.vue]: Then let's avoid the name :+1:

**Pull request merged**